### PR TITLE
Add a Pulumi 2.0 blog post

### PR DESCRIPTION
**Pull request by joeduffy**
_Note to self_: before merging, ensure

- [x] Links to `/` are replaced with `/superpowers`

### CODE REVIEW

**Comment by joeduffy** [comment]
Note: I'll replace the card with whatever @cnunciato comes up with, once it's ready.

File: content/blog/pulumi-2-0/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+## What is Pulumi?
+
+Pulumi's approach to modern infrastructure as code is unique:
+
+* **Real Languages**. Use the best languages and tools to declare infrastructure and gain best-in-class productivity and engineering capabilities. Connect to existing communities and ecosystems and leverage world-class language and tooling innovation.
+
+* **Multi-Cloud**. Adopt consistent workflows across any cloud(s) &mdash; public, private, or hybrid &mdash; while still using the full breadth and depth of your target cloud's services.
+
+* **Modern Architectures**. Although Pulumi works great for legacy workloads, it was born in a world of modern container, serverless, and cloud native architectures. Feel free to mix and match these approaches as you modernize over time.
```
Comment (spara) [inline content/blog/pulumi-2-0/index.md:29]:
```suggestion
* **Modern Architectures**. Although Pulumi works great for legacy workloads, it was born in a world of modern container, serverless, and cloud-native architectures. Feel free to mix and match these approaches as you modernize over time.
```

**Comment by joeduffy** [comment]
Also note: I can't link to the new pages yet, so you'll see `/` in place of `/superpowers` for now.

File: content/blog/pulumi-2-0/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+<img src="/images/mascot/pulumipus.svg" style="max-width: 240px; float: right; margin-top: -80px; padding: 4px;">
+
+We're also taking the opportunity to have a little fun with the launch and introducing a new mascot, _The Pulumipus_. You may have met them already at an event or at PulumiHQ, and we're happy to welcome them as the official mascot going forward.
+
+Expect The Pulumipus to pop up from time to time to help you out with your infrastructure superpowers!
+
+## Give it a Try Today
+
+Today is an exciting day for us, our customers, and our community &mdash; your passionate support and feedback shaped this release and we thank you.
```
Comment (spara) [inline content/blog/pulumi-2-0/index.md:65]:
```suggestion
Today is an exciting day for us, our customers, and our community &mdash; your passionate support and feedback shaped this release, and we thank you.
```

File: content/blog/pulumi-2-0/index.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+<div class="header-hero-actions mt-8 mb-8 text-center">
+    <a class="btn btn-lg mr-2" href="{{< relref "/docs/get-started" >}}">
+        Get Started
+    </a>
+    <a class="btn btn-lg btn-orange ml-2" href="{{< relref "/" >}}">
+        Learn More
+    </a>
+</div>
+
+Please tell your friends and colleagues &mdash; we will see you in [the Community Slack](https://slack.pulumi.com) where we look forward to hearing what you're doing with your new superpowers!
```
Comment (spara) [inline content/blog/pulumi-2-0/index.md:78]:
```suggestion
Please tell your friends and colleagues &mdash; we will see you in [the Community Slack](https://slack.pulumi.com), where we look forward to hearing what you're doing with your new superpowers!
```

File: content/blog/pulumi-2-0/index.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+The main theme of 2.0 is something we've taken to calling "superpowers." This isn't just a buzzword we came up with &mdash; it's something we keep hearing from happy end users:
+
+<div class="ml-8">
+{{< tweet 1169245701714468864 >}}
+</div>
+
+## What is Pulumi 2.0?
+
+Ater [shipping 1.0 last year]({{< relref "/blog/pulumi-1-0" >}}), we've been working hard at work helping customers to succeed at their modern cloud projects. This has taken us beyond the basics to include:
```
Comment (justinvp) [inline content/blog/pulumi-2-0/index.md:43]:
```suggestion
After [shipping 1.0 last year]({{< relref "/blog/pulumi-1-0" >}}), we've been working hard at work helping customers to succeed at their modern cloud projects. This has taken us beyond the basics to include:
```

File: content/blog/pulumi-2-0/index.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+The main theme of 2.0 is something we've taken to calling "superpowers." This isn't just a buzzword we came up with &mdash; it's something we keep hearing from happy end users:
+
+<div class="ml-8">
+{{< tweet 1169245701714468864 >}}
+</div>
+
+## What is Pulumi 2.0?
+
+After [shipping 1.0 last year]({{< relref "/blog/pulumi-1-0" >}}), we've been working hard at work helping customers to succeed at their modern cloud projects. This has taken us beyond the basics to include:
```
Comment (praneetloke) [inline content/blog/pulumi-2-0/index.md:45]:
```suggestion
After [shipping 1.0 last year]({{< relref "/blog/pulumi-1-0" >}}), we've been hard at work helping customers to succeed at their modern cloud projects. This has taken us beyond the basics to include:
```

or :point_down:

Comment (praneetloke) [inline content/blog/pulumi-2-0/index.md:45]:
```suggestion
After [shipping 1.0 last year]({{< relref "/blog/pulumi-1-0" >}}), we've been working hard helping customers to succeed at their modern cloud projects. This has taken us beyond the basics to include:
```

**Pull request merged**